hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/SimpleRegionNormalizer.java

```diff
@@ -502,6 +522,8 @@ private NormalizerConfiguration(final Configuration conf,
       mergeMinRegionCount = parseMergeMinRegionCount(conf);
       mergeMinRegionAge = parseMergeMinRegionAge(conf);
       mergeMinRegionSizeMb = parseMergeMinRegionSizeMb(conf);
+      cumulativePlansSizeLimitMb =
```

Comment:
i realize you need to keep this here for the shuffling stuff. that's fine, once the imports are updated to point at the constants in RegionNormalizerWorker.

but i dont think we should call `logConfigurationUpdated` here anymore, since the logger class will be SimpleRegionNormalizer even though the config is mostly for the worker. in another comment i ask you to add a log in RegionNormalizerWorker which I think can suffice for this config.

Reply:
Done